src/components/PassengerShowcaseForm.js

```diff
@@ -429,7 +435,7 @@ const PassengerShowcaseForm = () => {
           className={css.submitBtn}
           onClick={onSubmit}
           variant="purple"
-          disabled={submitted}
+          disabled={submitted !== 'not-submitted'}
```

Comment:
Nit-pick but I find it hard to parse cognitively with the double (triple?) negative.

Maybe renaming the variable to `submissionState` and finding an alternate name for `not-submitted` state that doesn't contain a negative would help? What about `undefined`, would that read better?

```ts
submissionState === 'submitted'

disabled = !!submissionState
```